skimage/feature/corner.py

```diff
@@ -915,16 +916,35 @@ def corner_subpix(image, corners, window_size=11, alpha=0.99):
 
 def corner_peaks(image, min_distance=1, threshold_abs=None, threshold_rel=None,
                  exclude_border=True, indices=True, num_peaks=np.inf,
-                 footprint=None, labels=None, *, num_peaks_per_label=np.inf):
+                 footprint=None, labels=None, *, num_peaks_per_label=np.inf,
+                 p=np.inf):
     """Find corners in corner measure response image.
 
     This differs from `skimage.feature.peak_local_max` in that it suppresses
     multiple connected peaks with the same accumulator value.
 
     Parameters
     ----------
+    image : ndarray
+        Input image.
+    min_distance : int, optional
+        The minimum distance seperating peaks. Use the ``p`` argument
+        to set the Minkowski p-norm defining the distance.
     * : *
         See :py:meth:`skimage.feature.peak_local_max`.
+    p : float
```

Comment:
Could we call this `p` as `p_norm` instead? I think we would benefit from a more descriptive name here.